packages/graphics/src/Graphics.js

```diff
@@ -33,6 +33,11 @@ let defaultShader = null;
  * The Graphics class contains methods used to draw primitive shapes such as lines, circles and
  * rectangles to the display, and to color and fill them.
  *
+ * Note that because Graphics can share a GraphicsGeometry with other instances,
+ * it is necessary to call `destroy()` to properly dereference the underlying
+ * GraphicsGeometry and avoid a memory leak. Alternatively, keep using the same
+ * Graphics instance and call `clear()` between frames.
+ *
```

Comment:
Looks good, the only comment here is “frames” is confusing. Might suggest “redraws” instead. Don’t want people to call clear between every tick or requestAnimationFrame if they don’t need to.

Reply:
Fixed 502312955fbff9c15a849ab51b5a93155ad8e4e7